AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/annotations/EBean.java

```diff
@@ -110,6 +110,22 @@
 		 */
 		Default, //
 
+		/**
+		 * A new instance of the bean is created the first time it is needed in an
```

Comment:
A new instance of the bean is created the first time it is needed in an
Activity scope, it is then retained and the same instance is always returned
from within the same Activity. Different Activities hold different copies of
the bean.